app/controllers/campaigns_controller.rb

```diff
@@ -1,15 +1,23 @@
 class CampaignsController < ApplicationController
 
   def new
-    @campaign = Campaign.new
     @organisation = Organisation.find(params[:organisation_id])
```

Comment:
Can refacto in a private functions as you use it in new and create

Reply:
i don't think so because you can't get params unless you're inside the method